packages/docusaurus-theme-classic/src/theme/CodeBlock/styles.module.css

```diff
@@ -29,6 +28,7 @@
   margin: 0;
   padding: 0;
   border-radius: 0;
+  opacity: 1;
```

Comment:
is this really necessary? What's the goal?

Reply:
You are right, was not needed! Have removed this now 😄